src/Select/common/SelectWrapper.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react'
+import propTypes from 'prop-types'
+import { InputWrapper } from './InputWrapper.js'
+import { MenuWrapper } from './MenuWrapper.js'
+
+// Keycodes for the keypress event handlers
+const ESCAPE_KEY = 27
+const SPACE_KEY = 32
+const UP_KEY = 38
+const DOWN_KEY = 40
+
+export class SelectWrapper extends Component {
```

Comment:
Ok, I've renamed it to Select. We 're now exporting:

- Select (just implements open/closing)
- SingleSelect 
- SingleSelectField
- SingleSelectOption
- MultiSelect
- MultiSelectField
- MultiSelectOption

I think we shouldn't export `Select` for now. It's a very basic component and to make use of it you'll also need the related Input and Menu components. I think it would be good to hold off on publishing all of that and let the abstractions settle a bit. Then we can expose the internals, if necessary, in a minor version. Otherwise any changes to the abstractions would warrant a major version bump.

---

> JSDocs are missing for the module/class

Yes, I'm holding off on updating those until we all agree on the abstractions/components/etc.